File: 005/5.Soru.cs
static long OBEB(long a, long b)
{
	while (b != 0)
	{
		long temp = b;
		b = a % b;
		a = temp;
	}
	return a;
}

static long EOK(long a, long b)
{
	return (a * b) / OBEB(a, b);
}

static void Main(string[] args)
{
	long sonuc = 1;
	for (long i = 1; i <= 20; i++)
	{
		sonuc = EOK(sonuc, i);
	}
	Console.WriteLine(sonuc); 
	Console.ReadLine();
}

File: 005/5.Soru.py
def OBEB(a,b):
    while b != 0:
        temp = b
        b = a % b
        a = temp
    return a

def EOK(a,b):
    return (a*b)//OBEB(a,b)

sonuc = 1
for i in range(1,21):
    sonuc = EOK(sonuc,i)
print(sonuc)
